common/src/main/java/tachyon/util/io/FileUtils.java

```diff
@@ -152,4 +152,17 @@ public static void createBlockPath(String path) throws IOException {
   public static void move(File from, File to) throws IOException {
     Files.move(from, to);
   }
+
+  /**
+   * Delete the file or directory
```

Comment:
If the directory is not empty, should this method succeed? If it can not succeed, `File.delete` can satisfy the requirements, but I think for this utility method, we should succeed?